rpc_jobs/rpc_aio.yml

```diff
@@ -3,6 +3,9 @@
     # Note: branch is the branch for periodics to build
     #       branches is the branch pattern to match for PR Jobs.
     series:
+      - kilo:
```

Comment:
Are you looking to add standalone kilo tests, separate from upgrades? I'm not clear from the commit message.